python/README.md

```diff
@@ -0,0 +1,27 @@
+# Python Arrow Flight Client Exmaple
+This lightweight Python client connects to the Dremio Arrow Flight server endpoint. This client requires the username and password for authentication. Developers can use admin or regular user credentials for authentication. This client can query any datasets in Dremio that are accessible by the provided Dremio user. By default, the hostname is `localhost` and the port is `32010`. Developers can change these default settings by providing the hostname and port as arguments when running the client. Moreover, the tls option can be provided to establish an encrypted connection. 
+> Note that: Trusted certificates must be provided when the tls option is enabled.
+
+## Instructions on using this Python client
+- Install and setup Python3 as `pyarrow` requires Python3
+- This client also requires `pyarrow` and `pandas`, consider one of dependency installation method below. We recommend using `conda` for its ease of use.
```

Comment:
Comma splice: two independent clauses spliced together with a comma. Options are 1) Split into two sentences or 2) Replace comma with a semicolon (;).

"consider one of dependency installation method below" 
- missing article - `the`- before 'dependency'
- pluralize 'method' -> 'methods'

Reply:
Addressed.